invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/detail.html

```diff
@@ -249,7 +249,11 @@ <h2>{{ add_description.type.title_l10n }} <span
         </dt>
         {% if field_value is string %}
           <dd>
-            {% set search_url = field_cfg.field | custom_fields_search(field_value|safe) %}
+            {% if field_cfg.field == 'code:codeRepository' %}
```

Comment:
major: I would avoid hardcoding this here, since `app-rdm` is not necessarily aware of the `code:` custom fields. Maybe a more "generic" way to do this would be to check the type of the custom field config and have some sort of attribute that tells us if it's an identifier, in which case we could be using the `pid_url` Jinja filter.